site/src/components/layout/rectangles-example-measure.js

```diff
@@ -114,15 +114,10 @@ var data = [
 //START
 var labelPadding = 2;
 
-// the container component is used to add padding around a text label
-var label = fc.tool.container()
+// a component which renders the labels
+var label = fc.layout.textLabel()
```

Comment:
The naming seems forced to fit in with `fc.layout.label`, not that I can think of a better name...